templates/al2/provisioners/install-worker-accelerator-support.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+if [[ ! "$ENABLE_ACCELERATOR" =~ ^(nvidia|neuron)$ && "$ENABLE_EFA" != "true" ]]; then
+  echo "Skipping worker accelerator support - pciutils and oci-add-hooks"
+  exit 0
+fi
```

Comment:
I'd be ok running this if either `ENABLE_ACCELERATOR` or `ENABLE_EFA` are set
```suggestion
if [[ -z "$ENABLE_ACCELERATOR" ]] &&  [[ "$ENABLE_EFA" != "true" ]]; then
  echo "Skipping worker accelerator support - pciutils and oci-add-hooks"
  exit 0
fi
```

Reply:
IMO there's no value in over-complicating the expression, whenever I can write a simpler script to do the same, I do that. I'll leave the decision to you though